src/site/content/en/blog/appcache-removal/index.md

```diff
@@ -0,0 +1,222 @@
+    </td>
+    <td>Chrome 50 (April 2016)
+    </td>
+    </tr>
+    <tr>
+    <td><a href="https://groups.google.com/a/chromium.org/g/blink-dev/c/ANnafFBhReY/m/1Xdr53KxBAAJ?pli=1">Removal from non-secure contexts</a>
+    </td>
+    <td>Chrome 70 (October 2018)
+    </td>
+    </tr>
+    <tr>
+    <td><a href="https://groups.google.com/a/chromium.org/g/blink-dev/c/FvM-qo7BfkI/m/0daqyD8kCQAJ">Deprecation in secure contexts</a>
+    </td>
+    <td>Chrome 79 (December 2019)
+    </td>
+    </tr>
+    <tr>
+    <td><a href="https://blog.chromium.org/2020/01/appcache-scope-restricted.html">AppCache scope restriction</a>
+    </td>
+    <td>Chrome 80 (February 2020)
+    </td>
+    </tr>
+    <tr>
+    <td>"Reverse" origin trial begins
+    </td>
+    <td>Chrome 84 (<a href="https://chromiumdash.appspot.com/schedule">estimated July 2020</a>)
+    </td>
+    </tr>
+    <tr>
+    <td><a href="https://groups.google.com/a/chromium.org/g/blink-dev/c/FvM-qo7BfkI/m/AvxoE6JpBgAJ">Removal from secure contexts</a>, except for those opted-in to the origin trial
+    </td>
+    <td>Chrome 85 (<a href="https://chromiumdash.appspot.com/schedule">estimated August 2020</a>)
+    </td>
+    </tr>
+    <tr>
+    <td>Complete removal from secure contexts for everyone, with completion of origin trial
+    </td>
+    <td>Chrome 90 (<a href="https://chromiumdash.appspot.com/schedule">estimated April 2021</a>)
+    </td>
+    </tr>
+  </table>
+</div>
+
+{% Aside %}
+This timeline applies to Chrome on **all platforms other than iOS**. There is also an adjusted timeline for AppCache used within an Android [WebView](https://developer.android.com/reference/android/webkit/WebView). For more info, see [The cross-platform story](#the-cross-platform-story) later in this post.
+{% endAside %}
+
+## Origin trial
+
+The timeline lists two upcoming milestones for removal. Beginning with Chrome 85, AppCache will no longer be available in Chrome by default. Developers who require additional time to migrate off of AppCache can [sign up](https://developers.chrome.com/origintrials/#/register_trial/1776670052997660673) for a "reverse" [origin trial](https://github.com/GoogleChrome/OriginTrials/blob/gh-pages/developer-guide.md) to extend the availability of AppCache for their web apps. The origin trial will start in Chrome 84 (in advance of the default removal in Chrome 85), and will be active through Chrome 89. Starting with Chrome 90, AppCache will be fully removed for everyone, even those who had signed up for the origin trial.
+
+{% Aside %}
+Why are we calling this a "reverse" origin trial? Normally, an origin trial allows developers to opt-in to early access to new functionality before it has shipped by default in Chrome. In this case, we're allowing developers to opt-in to using legacy technology even after it's been removed from Chrome, but only temporarily.
+{% endAside %}
```

Comment:
Love that you all explained this right here, that exact question came to my mind